packaging/examples/kafka/kraft/README.md

```diff
@@ -4,7 +4,8 @@ The examples in this directory demonstrate how you can use Kraft (ZooKeeper-less
 * The [`kafka.yaml`](kafka.yaml) deploys a Kafka cluster with one pool of _KRaft controller_ nodes and one pool of _KRaft broker_ nodes.
 * The [`kafka-ephemeral.yaml`](kafka-ephemeral.yaml) deploys a Kafka cluster with one pool of _KRaft controller_ nodes, one pool of _KRaft broker_ nodes and ephemeral storage.
 * The [`kafka-with-dual-role-nodes.yaml`](kafka-with-dual-role-nodes.yaml) deploys a Kafka cluster with one pool of KRaft nodes that share the _broker_ and _controller_ roles.
+* The [`kafka-single-node.yaml`](kafka-single-node.yaml) deploys a Kafka cluster with a single Kafka node that has both _broker_ and _controller_ roles.
 
-To use KRaft, ensure that the `KafkaNodePools` and `UseKRaft` feature gates are not disabled.
-Please note that ZooKeeper-less Apache Kafka is still under development and lacks some critical features.
-For instance, JBOD storage is not supported (you can specify `type: jbod` storage in Strimzi custom resources, but it should contain only a single volume).
+To use KRaft, ensure that the and `UseKRaft` feature gate is not disabled.
+Please note that ZooKeeper-less Apache Kafka is still under development and has some limitations.
+For instance, scaling of controller node or JBOD storage (you can specify `type: jbod` storage in Strimzi custom resources, but it should contain only a single volume) are not supported.
```

Comment:
```suggestion
For instance, scaling of controller node or JBOD storage are not supported.
You can specify `type: jbod` storage in Strimzi custom resources, but it should contain only a single volume.
```

Reply:
The JBOd note belongs specifically to the JBOD storage part, so I think it is better to have it directly there.

Reply:
Ok, but it's kind of weird to read. What about "If you still want to use JBOD, you can specify..."